packages/react/src/drafts/Tooltip/Tooltip.examples.stories.tsx

```diff
@@ -16,7 +16,7 @@ import {
 import {default as VisuallyHidden} from '../../_VisuallyHidden'
 
 export default {
-  title: 'Drafts/Components/Tooltip/Examples',
+  title: 'Components/TooltipV2/Examples',
```

Comment:
Pedantic and optional: `Tooltip v2` with a space please!

Reply:
No worries sounds good!

Reply:
Storybook tests (src/__tests__/storybook.test.tsx) failed with this change. 

```
Expected pattern: /Components\/\w+\/Features/
Received string:  "Components/Tooltip v2/Features"
```

I guess this was the pattern we decided for the names? Seems like the Dialog 1 has the same name as `DialogV2` - Would we want to re-visit the naming patterns? I also think we might have made that decision for regular components names, I doubt that it would include `V2` type of suffix.

Reply:
Oh ha! Nevermind then, not worth it!